app/uk/gov/hmrc/servicedependencies/testonly/IntegrationTestController.scala

```diff
@@ -78,12 +77,6 @@ class IntegrationTestController @Inject()(
         .map(_ => Ok("Done"))
     }
 
-  def addSlugDependencyConfigs =
-    Action.async(validateJson[Seq[DependencyConfig]]) { implicit request =>
-      Future.sequence(request.body.map(dependencyConfigRepo.add))
-        .map(_ => Ok("Done"))
-    }
-
   def deleteSbt =
```

Comment:
the endpoint was removed but this is still here?  Is that correct?

Reply:
Endpoint was removed accidentally, restored now.